docs/src/references/tedge-p11-server.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 🚧 tedge-p11-server
+tags: [Reference, Security, "PKCS #11", CLI]
+description: "An optional component for accessing PKCS #11 cryptographic tokens"
+draft: true
+---
+
+## Description
+
+**tedge-p11-server** is an optional component used by %%te%% for accessing PKCS #11 cryptographic
+tokens. It should be used when it's not possible to load the PKCS #11 dynamic module in `tedge`
+directly at runtime, e.g. when running a statically linked %%te%% on musl or when `tedge` doesn't
+have the permission to view/access cryptographic tokens.
```

Comment:
```suggestion
have the permission to view/access cryptographic tokens, because running a container.
```

Reply:
fixed in 5eaf881bc56b35d95695649d61027be0cb3f74ec